dedibox-network/domains/how-to/enable-dnssec.mdx

```diff
@@ -5,16 +5,12 @@ meta:
 content:
   h1: How to enable DNSSEC
   paragraph: This page explains how to enable DNSSEC
-steps:
-  - step: TODO
-    type: HowToStep
-    url: https://www.scaleway.com/en/docs/dedibox-network/domains/how-to/enable-dnssec
 dates:
   validation: 2021-05-26
   posted: 2021-05-26
 ---
 
-Since its design in 1983, DNS has been vulnerable to attacks. Attackers are able to falsify responses to DNS queries which allow them to redirect end users to web sites under their own control. In response to these threats, DNSSEC was deployed. DNSSEC cryptographically ensures that DNS content cannot be modified from its source without being detected. It works by digitally signing each DNS record so that any tampering of that record can be detected.
+Since its design in 1983, DNS has been vulnerable to attacks. Attackers are able to falsify responses to DNS queries which allow them to redirect end users to web sites under their own control. In response to these threats, [DNSSEC](https://www.icann.org/resources/pages/dnssec-what-is-it-why-important-2019-03-05-en) was deployed. DNSSEC cryptographically ensures that DNS content cannot be modified from its source without being detected. It works by digitally signing each DNS record so that any tampering of that record can be detected.
```

Comment:
```suggestion
Since its design in 1983, DNS has been vulnerable to attacks. Attackers are able to falsify responses to DNS queries which allow them to redirect end users to websites under their own control. In response to these threats, [DNSSEC](https://www.icann.org/resources/pages/dnssec-what-is-it-why-important-2019-03-05-en) was deployed. DNSSEC cryptographically ensures that DNS content cannot be modified from its source without being detected. It works by digitally signing each DNS record so that any tampering of that record can be detected.
```